api/v1alpha4/conditions.go

```diff
@@ -25,34 +25,34 @@ const (
 const (
 	// ClusterCategoryCreatedCondition indicates the status of the category linked to the NutanixCluster
 	ClusterCategoryCreatedCondition capiv1.ConditionType = "ClusterCategoryCreated"
-	ClusterCategoryCreationFailed                        = "ClusterCategoryCreationFailed"
+	ClusterCategoryCreationFailed   string               = "ClusterCategoryCreationFailed"
```

Comment:
This seems a bit unnecessary: https://go.dev/play/p/OXMPJYIpfW9

Reply:
Hmm. Linter was suggesting following kind of error: 
api/v1beta1/conditions.go:39:2: SA9004: only the first constant in this group has an explicit type (staticcheck)
    VMProvisionedCondition  capiv1.ConditionType = "VMProvisioned"

Description of SA9004 error type: https://staticcheck.io/docs/checks#SA9004

Reply:
Yes. This seems to be working. Adding  a space between the  lines removes the error given by linter.